.github/ISSUE_TEMPLATE

```diff
@@ -14,4 +13,4 @@ Any other information you want to share that is relevant to the issue being repo
 #### Technical details:
 
 * sendgrid-php Version: master (latest commit: [commit number])
-* PHP Version: 5.x
\ No newline at end of file
+* PHP Version: 7.x
```

Comment:
We also support PHP 5.6, could you please add that?

Reply:
Yeah I  can add that too, and you should also keep the php 7 version too with you.